Assets/MRTK/SDK/Experimental/ScrollingObjectCollection/ScrollingObjectCollection.cs

```diff
@@ -1562,63 +1609,92 @@ private float SafeDivisionFloat(float numerator, float denominator)
 
         /// <summary>
         /// Checks visibility of scroll content by iterating through all content renderers and colliders.
+        /// All inactive content objects and colliders are reactivated during visibility restoration. 
         /// </summary>
-        private void HideItems()
+        private void ManageVisibility(bool isRestoring = false)
```

Comment:
Does isRestoring actually mean "isRestoringVisibility"? Not suggesting a rename, just trying to understand word -> concept mappings